src/model/metric.py

```diff
@@ -102,6 +99,7 @@ def update(self, data, output):
         return None
 
     def finalize(self):
+        fid_score = import_given_path("fid_score", os.path.join(lib_path, 'pytorch_fid/fid_score.py'))
```

Comment:
We should import the module only once.
Better to initialize it as a class static attribute?